packages/components/toaster/src/utils/getToasterTypesStyles.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTheme } from '@douro-ui/react';
```

Comment:
Missing the unit test

Reply:
For this function, unit tests weren't  created because it only utilizes colors. It’s important to remind that we face issues when testing styles for colors, as Jest does not correctly identify the colors, leading to errors.